### PR TITLE
BER-FTWD-karl

### PR DESCRIPTION
**Pull request by karlkristopher**

### CODE REVIEW

**Comment by karlkristopher** [comment]
A couple final minor fixes.

File: src/movies.js [hunk 1]
```diff
@@ -2,14 +2,124 @@
...
+          return 0;
+    });
+    
+    const byYear = alpha.sort(function compare(a, b) {
+        return a.year - b.year;
+    });
+
+    return byYear;
+
+};
```
Comment (Jeff-Moraes) [inline src/movies.js]: nicely done

File: src/movies.js [hunk 2]
```diff
@@ -2,14 +2,124 @@
...
+
 // Iteration 3: Alphabetic Order - Order by title and print the first 20 titles
 
+function orderAlphabetically (arr) {
+    const alpha = [...arr].map(function(movie){
+        return movie.title;
+    }).sort();
+    return alpha.slice(0, 20);
+
+};
```
Comment (Jeff-Moraes) [inline src/movies.js]: Well done! short and clean code.

File: src/movies.js [hunk 3]
```diff
@@ -2,14 +2,124 @@
...
+    })
+
+
+    if (dramaMovies.length === 0) return 0;
+
+    const dramaSum = dramaMovies.reduce(function(sum, movie){
+        return sum + movie.rate;
+    }, 0)
+
+    return Math.round((dramaSum / dramaMovies.length) * 100) / 100
```
Comment (Jeff-Moraes) [inline src/movies.js]: nicely done. You could have also do
`return Number((dramaSum / dramaMovies.length).toFixed(2));`

**Pull request closed without merging**